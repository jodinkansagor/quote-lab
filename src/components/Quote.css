@import url('https://fonts.googleapis.com/css?family=Roboto+Condensed&display=swap');

.quoteBubble {
  width: 400px;

  h4 {
    font-family: 'Roboto Condensed', sans-serif
  }
}