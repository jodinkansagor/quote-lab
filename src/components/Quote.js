import React from 'react';
import PropTypes from 'prop-types';

const Quote = ({ text, source }) => (
  <>
    <h2>&quot;{text}&quot; -- {source}</h2>
  </>
);

Quote.propTypes = {
  text: PropTypes.string.isRequired,
  source: PropTypes.string.isRequired
};

export default Quote;
