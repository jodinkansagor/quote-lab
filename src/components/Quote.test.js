import React from 'react';
import { shallow } from 'enzyme';
import Quote from './Quote';

describe('Quote component', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<Quote text="A QUOTE" source="My MOM" />);
    expect(wrapper).toMatchSnapshot();
  })
})