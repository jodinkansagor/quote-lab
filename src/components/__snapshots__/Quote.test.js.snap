// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component matches a snapshot 1`] = `
<Fragment>
  <h2>
    "
    A QUOTE
    " -- 
    My MOM
  </h2>
</Fragment>
`;
